### ISSUE TITLE
when shuffling labels we have to mention array dimension correctly

### ISSUE THREAD

**Issue by dsadulla** [rstudio/keras#266]
https://github.com/rstudio/keras/blob/7abceabaeb3071a6b8813018d7119d9dde3a74d5/vignettes/examples/pretrained_word_embeddings.R#L116

`labels <- labels[indices,]` returns `Error in labels[indices, ] : incorrect number of dimensions`

`labels <- labels[indices,,]` runs as intended

Similar issue at the following lines as well:

https://github.com/rstudio/keras/blob/7abceabaeb3071a6b8813018d7119d9dde3a74d5/vignettes/examples/pretrained_word_embeddings.R#L120

Should be `y_train <- labels[-(1:num_validation_samples),,] `

https://github.com/rstudio/keras/blob/7abceabaeb3071a6b8813018d7119d9dde3a74d5/vignettes/examples/pretrained_word_embeddings.R#L122

Should be `y_val <- labels[1:num_validation_samples,,] `

Please correct it :)

**Comment by dsadulla** [comment]
This issue happened because I erroneously ran line `labels <- labels[indices,] ` twice and hence the dimension exploded.
